plugins/windows/src/main/java/net/java/games/input/DirectInputEnvironmentPlugin.java

```diff
@@ -138,6 +138,12 @@ public final Controller[] getControllers() {
 		return controllers;
 	}
 
+	@Override
+	public void release() {
+		// Not yet implemented!
```

Comment:
It _may_ be worth it to log a warning in each of the unimplemented `release` methods. Some potential benefits:

1. Devs/Users will clearly be warned that something may not be working as expected.
2. Devs may get the hint that this library could use some help in implementing them.

Reply:
That makes sense, considered just throwing an unimplemented exception, but logging works too 👌